build.assets/tooling/lib/github/workflows.go

```diff
@@ -92,6 +89,22 @@ func ListWorkflowRuns(ctx context.Context, actions WorkflowRuns, owner, repo, pa
 		listOptions.Page = resp.NextPage
 	}
 
+	return allRuns, nil
+}
+
+// ListWorkflowRunIds returns a set of RunIDs, representing the set of all for
+// workflow runs created since the supplied start time.
+func ListWorkflowRunIds(ctx context.Context, actions WorkflowRuns, owner, repo, path, branch string, since time.Time) (RunIDSet, error) {
```

Comment:
```suggestion
func ListWorkflowRunIDs(ctx context.Context, actions WorkflowRuns, owner, repo, path, branch string, since time.Time) (RunIDSet, error) {
```